apps/full-site-editing/full-site-editing-plugin/homepage-posts/index.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Hompage posts file.
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE;
+
+define( 'NEWSPACK_BLOCKS__BLOCKS_DIRECTORY', 'dist/' );
+define( 'NEWSPACK_BLOCKS__PLUGIN_DIR', plugin_dir_path( __FILE__ ) );
+define( 'NEWSPACK_BLOCKS__VERSION', '1.0.0-alpha.17' );
+
+/**
+ * Filters block name.
+ *
+ * @param string $name Block name.
+ * @return string
+ */
+function homepage_posts_block_name( $name ) {
+	if ( 'newspack-blocks/homepage-articles' === $name ) {
+		return 'a8c/homepage-posts';
+	}
+	return $name;
+}
+add_filter( 'newspack_blocks_block_name', __NAMESPACE__ . '\homepage_posts_block_name' );
+
+/**
+ * Filters block arguments for `register_block_type()`.
+ *
+ * @param array  $args Arguments to `register_block_type()`.
+ * @param string $name Block name.
+ * @return array
+ */
+function homepage_posts_block_args( $args, $name ) {
+	if ( 'newspack-blocks/homepage-articles' !== $name ) {
```

Comment:
⚠️ When syn with `master`, the block name for NHA, on this point, doesn't contain the namespace. It means the value of `$name` is `homepage-articles` which means that the block never will be registered.

Reply:
Updated here: https://github.com/Automattic/wp-calypso/pull/37923/commits/d23394a3eb9130b360a24dd6bf792d381af309c7